### ISSUE TITLE
About the loss to train the Style-Complement Module G

### ISSUE THREAD

**Issue by bregmangh** [BUserName/Learning_to_diversify#1]
Thanks for your nice work! I have a question about the following code in train.py: 
(dist - self.args.beta * div).backward()
As I understand it, it should look like the following:
(dist + self.args.beta * div).backward()
Is it?

**Comment by BUserName** [comment]
Hey there,
Thank you so much for pointing this out!  The symbol was flipped when I was on a cross-entropy experiment (line 155) for the rebuttal and I didn't realize I forgot to change it back.

Updating the repo.
